### PR TITLE
[zh-cn]sync validating-admission-policy-binding-list-v1beta1.md

### PR DESCRIPTION
**Pull request by howieyuen**

### CODE REVIEW

File: content/zh-cn/docs/reference/kubernetes-api/other-resources/validating-admission-policy-binding-list-v1beta1.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+api_metadata:
+  apiVersion: "admissionregistration.k8s.io/v1beta1"
+  import: "k8s.io/api/admissionregistration/v1beta1"
+  kind: "ValidatingAdmissionPolicyBindingList"
+content_type: "api_reference"
+description: ""
+title: "ValidatingAdmissionPolicyBindingList v1beta1"
+weight: 1
+auto_generated: true
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/kubernetes-api/other-resources/validating-admission-policy-binding-list-v1beta1.md:10]: Should we remove this line? It is manually copied currently

File: content/zh-cn/docs/reference/kubernetes-api/other-resources/validating-admission-policy-binding-list-v1beta1.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+<!--
+The file is auto-generated from the Go source code of the component using a generic
+[generator](https://github.com/kubernetes-sigs/reference-docs/). To learn how
+to generate the reference documentation, please read
+[Contributing to the reference documentation](/docs/contribute/generate-ref-docs/).
+To update the reference content, please follow the 
+[Contributing upstream](/docs/contribute/generate-ref-docs/contribute-upstream/)
+guide. You can file document formatting bugs against the
+[reference-docs](https://github.com/kubernetes-sigs/reference-docs/) project.
+-->
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/kubernetes-api/other-resources/validating-admission-policy-binding-list-v1beta1.md:22]: Ditto

File: content/zh-cn/docs/reference/kubernetes-api/other-resources/validating-admission-policy-binding-list-v1beta1.md (hunk 1 above)
Comment (howieyuen) [reply content/zh-cn/docs/reference/kubernetes-api/other-resources/validating-admission-policy-binding-list-v1beta1.md:10]: yes, thanks for noticing

**Comment by windsonsea** [comment]
Thanks
/lgtm
/approve

**Pull request merged**